Conceptpower+Spring/src/main/java/edu/asu/conceptpower/servlet/web/ConceptEditController.java

```diff
@@ -258,6 +267,47 @@ public ModelAndView confirmEdit(@PathVariable("id") String id, HttpServletReques
         return new ResponseEntity<String>(buildJSON(synonymList, false, true), HttpStatus.OK);
     }
 
+    @RequestMapping(method = RequestMethod.GET, value = "conceptEdit/conceptSearch")
+    public @ResponseBody ResponseEntity<Object> searchConcept(@RequestParam("concept") String concept,
+            @RequestParam("pos") String pos) throws IllegalAccessException, LuceneException {
+
+        List<ConceptEntryWrapper> foundConcepts = null;
+
+        if (!concept.trim().isEmpty()) {
+
+            ConceptEntry[] found = null;
+
+            if (indexService.isIndexerRunning()) {
+                return new ResponseEntity<Object>("Indexer is running. Try again after sometime.",
+                        HttpStatus.SERVICE_UNAVAILABLE);
+            }
+
+            try {
+                found = conceptManager.getConceptListEntriesForWord(concept, pos,
+                        Constants.WORDNET_DICTIONARY);
+            } catch (IndexerRunningException ie) {
+                return new ResponseEntity<Object>("Indexer is running. Try again after sometime.",
+                        HttpStatus.SERVICE_UNAVAILABLE);
+            }
+
+            foundConcepts = new CopyOnWriteArrayList<>(wrapperCreator.createWrappers(found));
+        }
+        
+        // Remove CCP concepts. 
+        for(ConceptEntryWrapper wrapper : foundConcepts) {
+            System.out.println(wrapper.getEntry().getId());
+            if (wrapper.getEntry().getId().startsWith(Constants.CONCEPT_PREFIX)) {
```

Comment:
@jdamerow  Yes there is a class called ConceptTypesService but that class has been added as part of CCP-163. May be i will create this PR once that is merged